### ISSUE TITLE
About L1 Distance on TaichiHD Dataset

### ISSUE THREAD

**Issue by WeilunWang** [AliaksandrSiarohin/first-order-model#189]
Hello. I am very confused why I used the code and the pretrained model  you provided test on taichiHD. I get a lower L1 Distance(meaning better quality) than the result you given on your paper.  I compared the video requested as the test set in the csv file with my test set many times and found that there is no problem. Testing process use the code you provided. So, is it because you made some optimizations after submitting your paper?

**Comment by AliaksandrSiarohin** [comment]
Yes, the provided checkpoint is trained for more time, some 1.5x.

**Comment by WeilunWang** [comment]
OK. Thank you.

**Comment by WeilunWang** [comment]
By the way, I want to ask that what AKD, MKR, AED is in your paper. I can't find them on your paper or on the Internet.

**Comment by AliaksandrSiarohin** [comment]
See https://github.com/AliaksandrSiarohin/pose-evaluation
